api/hasura/images.ts

```diff
@@ -0,0 +1,45 @@
+import { VercelResponse } from '@vercel/node';
+import sharp from 'sharp';
+
+import { MAX_IMAGE_BYTES_LENGTH } from '../../src/lib/images';
+
+export function parseBase64Image(
+  res: VercelResponse,
+  imageDataBase64: string
+): Buffer | undefined {
+  // base64 decode the provided image data
+  const imageBytes = Buffer.from(imageDataBase64, 'base64');
```

Comment:
Can we abstract this into a subfunction that lives in src/lib and rename this function `parseBase64ImageWithResponse`? It'd be fun to make this outer function more generic together, but I think keeping the outer function the interface as-is for now is fine

Reply:
The reason it's not in src/lib is because it refers to VercelResponse, which seemed inappropriate for the FE, since this is all backend code.

Reply:
moving this and renaming, i'm tripping